eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -25,34 +27,38 @@ jobs:
         displayName: 'Install dependencies'
 
       - script: |
+          mkdir $(Build.ArtifactStagingDirectory)/iphonesimulator/
           set -o pipefail
           xcodebuild archive \
                      -workspace AzureSDK.xcworkspace \
                      -scheme ${{parameters.BuildScheme}} \
                      -sdk iphonesimulator \
                      -configuration Release \
-                     -archivePath $(Build.ArtifactStagingDirectory)/AzureSDK-iphonesimulator.xcarchive \
+                     -archivePath $(Build.ArtifactStagingDirectory)/iphonesimulator/AzureSDK-iphonesimulator.xcarchive \
                      SKIP_INSTALL=NO | xcpretty -c
         displayName: 'Build and archive libraries for release [iphonesimulator]'
 
-      - publish: $(Build.ArtifactStagingDirectory)/AzureSDK-iphonesimulator.xcarchive
-        displayName: 'Publish release archive [iphonesimulator]'
-        artifact: 'archive-release-iphonesimulator'
+      - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
```

Comment:
```suggestion
      - template: /eng/common/pipelines/templates/steps/publish-1es-artifact.yml
```

Reply:
Or is there a specific reason for the old one?

Reply:
It was missed I have since caught it.